packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -40,6 +40,22 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownPropsAndState, DropdownVa
     }),
   }),
 
+  clearIndicator: ({ variables: v }): ICSSInJSStyle => ({
```

Comment:
I have a bad feeling about this, actually it duplicates styles for `toggleIndicator`. In the same time, we should allow to customize `clearIndicator` separately.

Move these styles to a separate function? 🤔